### DISCUSSION TITLE
请问一下名门翠园是被救了吗？

### DISCUSSION THREAD

**Discussion by Wtt0413** [WeNeedHome/SummaryOfLoanSuspension#921] [General]
好像有消息说名门翠园被救了，好像说要复工，但是不确定消息的准确性，想询问一下相关了解人士，谢谢！

**Comment by richardo876** [comment]
听其言，观其行。很多地方政府一开始的时候还对很多烂尾楼业主承诺复工了，结果都是睁眼说瞎话。

**Reply by Wtt0413** [reply]
嗯嗯，没错，谢谢，我会继续关注的！
